src/index.css

```diff
@@ -1,13 +1,44 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 20px;
+  font-family: "Poppins";
+  background: var(--secondary);
+  height: 1000px;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+header nav {
+  display: flex;
+  gap: 16px;
+  justify-content: space-around;
+  max-width: 1200px;
+  margin-bottom: 40px;
+}
+
+.item-img {
+  height: 75px;
+  width: 75px;
+}
+
+.item-card {
+  border-radius: 3px;
+  border-style: solid;
+}
+
+.itemListWrapper {
+  list-style: hidden;
+}
+
+@media (max-width: 1250px) {
```

Comment:
Your header isn't centered when viewed full screen. Consider changing the max-width on the media query 👍